wordpress-mobile/WordPress-iOS
# [Stats Refresh] Countries Map

Refs. #11852

This PR introduces the Countries Map, added to the _Countries_ section.

![IMG_3323](https://user-images.githubusercontent.com/912252/59519423-bc444800-8ebf-11e9-97f4-c797bc4a9d88.PNG)

![IMG_3324](https://user-images.githubusercontent.com/912252/59519424-bcdcde80-8ebf-11e9-8b29-1b8d02f5f517.PNG)


## To test:
• Open stats from a selected blog
• Select one of the period tab DWMY
• Scroll until you see the Countries section
• If there's no data, the map is hidden
• Click on `View More` to open the Stats detail and see the Map with the whole countries list

Update release notes:
